libs/java/service/src/main/java/ca/bc/gov/tno/services/converters/Settings.java

```diff
@@ -0,0 +1,23 @@
+package ca.bc.gov.tno.services.converters;
+
+/**
+ * Settings static class, provides configuration settings.
+ */
+public class Settings {
+  /**
+   * Default DateTime format. 2022-04-11T18:56:55.253908Z
+   */
+  public final static String dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSX";
+
+  /**
+   * Handle multiple format types.
+   * This is ridiculous, but java is horrible.
+   */
+  public final static String[] dateTimeFormats = new String[] {
```

Comment:
Had to add these to handle the various errors occurring during deserialization.  Java can't handle dates... it's embarrassing!